client/src/views/asset-list.js

```diff
@@ -15,30 +40,109 @@ export default ({ assetMap, t, ...S }) => {
           <h1>{t`Registered assets`}</h1>
         </div>
       </div>
-
       <div className="container">
-        { !assets.length ? <p>{t`No registered assets`}</p>
-        : <div className="assets-table">
+        { !assets ? <div className="load-more-container"><img src="img/Loading.gif" /></div>
+          : <div className="assets-table">
               <div className="assets-table-row header">
-                <div className="assets-table-cell">{t`Name`}</div>
-                <div className="assets-table-cell ticker">{t`Ticker`}</div>
-                <div className="assets-table-cell">{t`Issuer domain`}</div>
-                <div className="assets-table-cell">{t`Asset ID`}</div>
+                <a href={`/assets/registry?sort_field=name&sort_dir=${encodeURIComponent(sort_dir === 'asc' ? 'desc' : 'asc')}`} 
```

Comment:
Also no leading `/` (here and in the two links below)

Reply:
changes have been made